src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnectionBuilder.cs

```diff
@@ -25,7 +25,7 @@ public void AddSetting<T>(string name, T value)
 
         public bool TryGetSetting<T>(string name, out T value)
         {
-            value = default(T);
+            value = default;
```

Comment:
Does this work for all `T`s that we care about?

Reply:
It should work just the same.